packages/react-components/react-theme-sass/.babelrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
most of the autogenerated stuff is not useful and will never be used as long as this package just exports sass